Frameworks/UIKit.Xaml/TextField.xaml.cpp

```diff
@@ -0,0 +1,119 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+// clang-format does not like C++/CX
+// clang-format off
+#include "pch.h"
+#include "TextField.xaml.h"
+
+#include "ObjCXamlControls.h"
+
+using namespace Microsoft::WRL;
+using namespace Platform;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+
+namespace UIKit {
+namespace Xaml {
+
+TextField::TextField() {
+    InitializeComponent();
+}
+
+// Accessor for our Layer content
+Image^ TextField::LayerContent::get() {
+    if (!_content) {
+        _content = ref new Image();
+        _content->Name = "LayerContent";
+
+        // Layer content is behind any sublayers
+        textFieldCanvas->Children->InsertAt(0, _content);
+    }
+
+    return _content;
+}
+
+// Accessor for our Layer content
+bool TextField::HasLayerContent::get() {
+    return _content != nullptr;
+}
+
+// Accessor for our SublayerCanvas
+Canvas^ TextField::SublayerCanvas::get() {
+    return textFieldCanvas;
+}
+
+// Accessor for the LayerProperty that manages the BorderBrush of this TextField
+Private::CoreAnimation::LayerProperty^ TextField::GetBorderBrushProperty() {
+    return nullptr;
+}
+
+// Accessor for the LayerProperty that manages the BorderThickness of this TextField
+Private::CoreAnimation::LayerProperty^ TextField::GetBorderThicknessProperty() {
+    return nullptr;
+}
+
+void TextField::SetSecureTextEntryValue(bool secureTextEntry) {
```

Comment:
for the same reason.SecureTextEntry is what ref platform using.

---
In reply to: [110059624](https://github.com/Microsoft/WinObjC/pull/2411#discussion_r110059624) [](ancestors = 110059624)